daml-lf/parser/src/main/scala/com/digitalasset/daml/lf/testing/parser/ExprParser.scala

```diff
@@ -324,7 +325,12 @@ private[parser] class ExprParser[P](parserParameters: ParserParameters[P]) {
     "BIGNUMERIC_TO_TEXT" -> BBigNumericToText,
   )
 
-  private lazy val experimental: Parser[Expr] =
+  private lazy val eCallInterface: Parser[ECallInterface] =
+    Id("icall") ~! `@` ~> fullIdentifier ~ id ~ expr0 ^^ { case ifaceId ~ name ~ body =>
```

Comment:
Why `icall` and not `call_interface`? Seems a bit odd to abbreviate this one.

Reply:
I will sneak  that in another PR.
What do you think about `call_method` ?

Reply:
That works, but personally I'd like it to match the AST. We can change the AST to  `ECallMethod` if you like?

Reply:
We could have a whole pass on naming before release.